hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/bucket/HoodieBucketIndex.java

```diff
@@ -138,27 +112,20 @@ public boolean isGlobal() {
 
   @Override
   public boolean canIndexLogFiles() {
-    return false;
+    return true;
```

Comment:
can directly append to log files?

Reply:
For consistent hashing index, yes. For simple bucket index, have to create the base file first (whose naming contains the bucket number)